test/rfqt_test.ts

```diff
@@ -137,7 +137,7 @@ describe(SUITE_NAME, () => {
                             {
                                 ...DEFAULT_RFQT_RESPONSE_DATA,
                                 responseData: { signedOrder },
-                            },
+                            } as any,
```

Comment:
We are just mocking a subset of the required data for the test so it's fine to coerce it to `any` here I think